src/shared/analytics/analytics.background.ts

```diff
@@ -149,6 +153,35 @@ function trackAppEvents({ account }: { account: Account }) {
     });
     sendToMetabase('add_custom_evm', params);
   });
+
+  emitter.on('globalPreferencesChange', (state, prevState) => {
+    onIdle(() => {
```

Comment:
I would like to see this as a separate named function, so in this common file it is clear what do we want from globalPreferencesChange event. But not critical